src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj

```diff
@@ -36,6 +36,8 @@
     <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.5.0.x86" PrivateAssets="All" />
     <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.6.0.x64" PrivateAssets="All" />
     <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.6.0.x86" PrivateAssets="All" />
+    <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.7.0.x64" PrivateAssets="All" />
+    <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.7.0.x86" PrivateAssets="All" />
```

Comment:
@dotnet/aspnet-build @danmoseley I know we release Microsoft.AspNetCore.AzureAppServices.SiteExtension to NuGet.org. But what are our requirements for the content of the bundle❔ For example, should the 2.1 (which are `netcoreapp2.1` only) and 5.0 bits be removed now and 3.1.x soon❔

Might also be nice to stop building the bundles in release/6.0 and earlier since the 7.0.x bundle includes everything. Of course, customer messaging will get interesting if we do that.